distribution/docker/Dockerfile

```diff
@@ -105,8 +106,12 @@ RUN mkdir /opt/druid/var /opt/shared \
 # Install iproute2 to get the ip command needed to set config of druid.host IP address
 # Command needed in druid.sh Line 140;
 RUN if [ ! -x "$(command -v ip)" ]; then \
-      apt update \
-      && apt install -y iproute2; \
+      if [ -x "$(command -v apt)" ]; then \
+        apt update \
+        && apt install -y iproute2; \
+      else \
+        apk add iproute2; \
```

Comment:
This can fail if command apk is not present.

Reply:
Currently, the images we support use either the `apk` or `apt` package manager. I think if there are any changes in the future to the base image and the package managers they support, we should update the Dockerfile accordingly rather than trying to accommodate those changes now.